sphinx/source/docs/user_guide/embed.rst

```diff
@@ -134,7 +134,11 @@ For example, to use version ``0.12.0``, including widgets support:
 
 .. note::
     You must provide the closing `</script>` tag. This is required by all
-    browsers and the page will typically not render without it.
+    browsers and the page will typically not render without it. You may also
+    encounter a situation in which the browser will not load your page due to
+    an "unsafe" script. In this case, you may need to replace the above URLs
+    with "https" versus "http". That is, CDN resources are also available
+    via HTTPS URLS
```

Comment:
@bryevdv thanks. I'll git it another shot. I was trying to squeeze it into the 'note', but really it could be it's own paragraph also.